src/app/exercise/exercises/IntervalExercise/intervalExercise.ts

```diff
@@ -131,15 +131,19 @@ export const intervalExercise = () => {
     getQuestion(settings: IntervalExerciseSettings): Exercise.Question<IntervalName> {
       const randomIntervalDescriptor: IntervalDescriptor = randomFromList(intervalDescriptorList.filter(intervalDescriptor => settings.includedAnswers.includes(intervalDescriptor.name)));
       const randomStartingNoteNumber: NoteNumber = _.random(range.lowestNoteNumber, range.highestNoteNumber - randomIntervalDescriptor.semitones);
-      const startNoteName = toNoteName(randomStartingNoteNumber);
-      const endNoteName = toNoteName(randomStartingNoteNumber + randomIntervalDescriptor.semitones);
+      let startNoteName = toNoteName(randomStartingNoteNumber);
+      let endNoteName = toNoteName(randomStartingNoteNumber + randomIntervalDescriptor.semitones);
+      let partToPlayArray = _.shuffle([startNoteName, endNoteName]);
+      if (partToPlayArray[0] !== startNoteName) {
+        [startNoteName, endNoteName] = [endNoteName, startNoteName];
+      }
```

Comment:
Nice! A small suggestion though:
```suggestion
      let lowNoteName = toNoteName(randomStartingNoteNumber);
      let highNoteName = toNoteName(randomStartingNoteNumber + randomIntervalDescriptor.semitones);
      let [startNoteName, endNoteName] = _.shuffle([startNoteName, endNoteName])
```

I feel like this approach is more elegant and more readable. Then you can just pass "[startNoteName, endNoteName]" to partToPlay.